src/aeso_pretty.erl

```diff
@@ -430,6 +430,7 @@ lc_bind(Let) ->
 bin_prec('..')   -> {  0,   0,   0};  %% Always printed inside '[ ]'
 bin_prec('=')    -> {  0,   0,   0};  %% Always printed inside '[ ]'
 bin_prec('@')    -> {  0,   0,   0};  %% Only in error messages
+bin_prec('|>')   -> {150, 150, 200};
```

Comment:
I just noticed that this was missing, do you think it's correct?

Reply:
Check if it behaves nice with other ops, such as

2+2 |> somefun

2::[] ++ [] |> somefun

Reply:
Do you know what the 3 numbers returned by the function mean?

Reply:
Check usage, but I would guess operator priority over pre-, in-, and suffix operators

Reply:
It looks like when using `infixl`, the first and second numbers are the same, and the third number is the next precedence. This looks correct.